Apps/Playground/Win32/App.cpp

```diff
@@ -105,11 +105,12 @@ namespace
         inputBuffer = std::make_unique<InputManager<Babylon::AppRuntime>::InputBuffer>(*runtime);
 
         runtime->Dispatch([width, height, hWnd](Napi::Env env) {
+#ifndef V8_USE_INSPECTOR
```

Comment:
Instead of gating entirely behind macros, could this be conditionally enabled at runtime? For node I believe they take like an `--inspect` flag or something to signal that the inspector support should be enabled.

---
In reply to: [580411539](https://github.com/BabylonJS/BabylonNative/pull/607#discussion_r580411539) [](ancestors = 580411539)

Reply:
Perhaps we can do both. I would suspect not everyone would want the overhead of V8 inspector in their production apps.

Reply:
I think that's reasonable, though what're your thoughts on the default behavior? If the runtime enablement is gated behind a flag, the overhead is only the binary size increase.  In that case I think if `NAPI_JAVASCRIPT_ENGINE=V8` is set, we should include it with an additional opt-out flag (`V8_NO_INSPECTOR`?) to strip the code for places where binary size matters.

Reply:
That's an option. Another option is that the V8 inspector is a separate static library dependency and if you choose not to bring it in, then it won't be included in the binary size.

Reply:
We don't currently have a way to pass information between the app and the runtime, I cleaned up the logic to reduce the usage of V8_USE_INSPECTOR to just here. You can conditionally include the inspector library when running CMake with `-D V8_INCLUDE_INSPECTOR=[ON|OFF]` (default is ON).

I've [filed this issue](https://github.com/BabylonJS/BabylonNative/issues/640) to track addition of an `AppRuntimeOptions` struct that we can use to communicate to the runtime to initialize the V8 inspector (assuming the static library is included), and then the App code here can just not polyfill the console if it chooses to initialize the V8 inspector.

Reply:
Hmm, I'm not sure having AppRuntimeOptions is right for this. It's specific to V8. Would a static option specific to V8 be okay? I don't see why anyone would want two AppRuntimes and have them behave differently in terms of V8 inspector support.

Reply:
Cleaned it up using a V8-specific header. Feels better: https://github.com/BabylonJS/BabylonNative/pull/607/commits/ac13b7d56bfd18ebbc9991475b41d88414e3d18e

And also hits @tymlipari's comment about specifying the port.